_docs/tasks/telemetry/fluentd.md

```diff
@@ -0,0 +1,405 @@
+---
+title: Logging with Fluentd
+
+overview: This task shows you how to configure Istio to log to a Fluentd daemon
+
+order: 60
+
+layout: docs
+type: markdown
+---
+{% include home.html %}
+
+This task shows how to configure Istio to create custom log entries
+and send them to a Fluentd daemon. At the end of this task, a new log
+stream will be enabled sending logs to an example Fluentd /
+Elasticsearch / Kibana stack.
+
+The [BookInfo]({{home}}/docs/guides/bookinfo.html) sample application is used
+as the example application throughout this task.
+
+## Before you begin
+* [Install Istio]({{home}}/docs/setup/) in your cluster and deploy an
+  application. This task assumes that Mixer is setup in a default configuration
+  (`--configDefaultNamespace=istio-system`). If you use a different
+  value, update the configuration and commands in this task to match the value.
+
+## Setup Fluentd
+
+In your cluster, you may already have a Fluentd DaemonSet running,
+such the add-on described
+[here](https://kubernetes.io/docs/tasks/debug-application-cluster/logging-elasticsearch-kibana/)
+and
+[here](https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/fluentd-elasticsearch),
+or something specific to your cluster provider. This is likely
+configured to send logs to an Elasticsearch system or logging
+provider.
+
+You may use these Fluentd daemons, or any other Fluentd daemon you
+have set up, as long as they are listening for forwarded logs, and
+Istio's Mixer is able to connect to them. In order for Mixer to connect to a
+running Fluentd daemon, you may need to add a
+[service](https://kubernetes.io/docs/concepts/services-networking/service/). The
+Fluentd configuration to listen for forwarded logs is:
+
+```
+<source>
+  type forward
+</source>
+```
+
+The full details of connecting Mixer to all possible Fluentd
```

Comment:
This is fine. Can we start with adding a GKE example (where we already have a fluentd daemon running) of config somewhere and add a TODO placeholder for IBM Cloud Container Service and OpenShift Origin (to match the k8s quickstart page: https://istio.io/docs/setup/kubernetes/quick-start.html)?

Reply:
Yes, I'll need to work on the proper config. I'm not sure where this would go, maybe something like: docs/setup/kubernetes/gke-logging.html Also, would this be using the stackdriver adapter instead of fluentd?